src/goto-symex/goto_symex.h

```diff
@@ -95,6 +96,8 @@ class goto_symext
   optionst options;
   symbol_tablet &new_symbol_table;
 
+  irep_idt mode;
+
```

Comment:
We haven't been doing this thus far, but: we could get into the good habit of adding (doxygen-style) documentation of such members. The reason I'm bringing it up is that the name "mode" is rather generic.